File: Backend/tranvel/src/main/java/com/ssafy/tranvel/dto/TokenDto.java
package com.ssafy.tranvel.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

// 로그인 성공 시, JWT를 반환하는 DTO

@Builder
@Data
@AllArgsConstructor
public class TokenDto {

    private String grantType; // prefix: Bearer
    private String accessToken;
    private String refreshToken;

}

File: Frontend/app/src/main/java/com/ssafy/tranvel/data/model/TokenDto.kt
package com.ssafy.tranvel.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class TokenDto(
    val grantType: String,
    val accessToken: String,
    val refreshToken: String
) : Parcelable